content/en/docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -273,9 +273,11 @@ which matches the behavior when this feature is disabled.
 
 ## Taint Nodes by Condition
 
-Version 1.8 introduces an alpha feature that causes the node controller to create taints corresponding to
-Node conditions. When this feature is enabled (you can do this by including `TaintNodesByCondition=true` in the `--feature-gates` command line flag to the scheduler, such as
-`--feature-gates=FooBar=true,TaintNodesByCondition=true`), the scheduler does not check Node conditions; instead the scheduler checks taints. This assures that Node conditions don't affect what's scheduled onto the Node. The user can choose to ignore some of the Node's problems (represented as Node conditions) by adding appropriate Pod tolerations.
+In version 1.12, `TaintNodesByCondition` feature is promoted to beta, so node controller automatically creates taints corresponding to
+Node conditions.
+> this feature can be disabled by including `TaintNodesByCondition=true` in the `--feature-gates` command line flag to the scheduler, such as 
+`--feature-gates=...,TaintNodesByCondition=false`).
+Similarly the scheduler does not check Node conditions; instead the scheduler checks taints. This assures that Node conditions don't affect what's scheduled onto the Node. The user can choose to ignore some of the Node's problems (represented as Node conditions) by adding appropriate Pod tolerations.
```

Comment:
> This assures that Node conditions don't affect what's scheduled onto the Node

We'd better to clarify that `TaintNodesByCondition` only taints node with `NoSchedule` effect. As `TaintBasedEviction` will taint `NoExecute`.